helm-charts/common/whisper/README.md

```diff
@@ -12,6 +12,37 @@ export ASR_MODEL_PATH="openai/whisper-small"
 helm install whisper whisper --set global.modelUseHostPath=${MODELDIR} --set ASR_MODEL_PATH=${ASR_MODEL_PATH}
 ```
 
+### Install the microservice in air gapped (offline) mode
+
+To run `whisper` microservice in an air gapped environment, users are required to pre-download the model `openai/whisper-small` to a shared storage.
+
+#### Use node-local directory
+
+Assuming the model data is shared using node-local directory `/mnt/opea-models`.
+
+```
+# On every K8s node, run the following command:
+export MODEL_DIR=/mnt/opea-models
+# Download model, assumes Python huggingface_hub[cli] module is already installed
+huggingface-cli download --cache-dir "${MODEL_DIR}" openai/whisper-small
+
+# On K8s master node, run the following command:
+# Install using Helm with the following additional parameters:
+helm install ... ... --set global.offline=true,global.modelUseHostPath=${MODEL_DIR}
```

Comment:
nit: too many dots. 3 is enough (...)

Reply:
FYI: added PR to remove them also from other docs for consistency: https://github.com/opea-project/GenAIInfra/pull/1151